src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -33,6 +33,13 @@
     private String expenseType;
     private String cashAdvanceKey;
     private String reportEntryId;
+    private Boolean reportEntryIsPersonalFlag;
```

Comment:
Is this boolean enough?  I heard I needed to look at the value "Personal/Non Reimbursable" in the CBCP row?  I don't know for certain, am asking.